README.md

```diff
@@ -0,0 +1,52 @@
+# Static Site Generator App Extension for Quasar, the Vue.js Framework
+
+## Warning
+This extension for Quasar is an early-stage extension and under active development. Use at your own risk.
+
+## Development/Testing
+If you're interested in assisting with testing or development, you can get started using the following steps:
+
+1. cd into your quasar directory or run
+    ````bash
+    quasar create test-ext && cd test-ext
+    ````
+1. Install a couple dependencies:
+    ````bash
+    yarn add --dev https://github.com/freddy38510/quasar-app-extension-ssg critters-webpack-plugin
```

Comment:
I really appreciate that your wrote this missing documentation, thanks you.

You should not add the dependency "critters-webpack-plugin" as it is already added in the extension. Like you could see in "package.json" this is a fork of Critters i made, not the original repository. The goal of this fork is to use Critters without webpack, after the static pages are generated.